Packs/FeedDHS/Integrations/DHS_Feed/test_data/data_from_DHS/Domain_data.json

```diff
@@ -0,0 +1,1827 @@
+{
```

Comment:
this file is the raw data.
the second file is after a small change (e.g. instead of `{ "test1" : {"test2": {"test3": "test4"}}}` it’s just `{"test3": "test4"}`).